app/assets/stylesheets/comments.scss

```diff
@@ -28,19 +41,124 @@
       border-left: 3px solid $blue;
       padding-left: 3px;
     }
+
+    .publisher-textarea-wrapper {
+      background-color: $white;
+      border: 1px solid $border-dark-grey;
+      border-radius: 3px;
+
+      &.with-attachments .photodropzone-container {
+        border-top: 1px dashed $light-grey;
+      }
+
+      .upload-error {
+        background-color: $red;
+        border-top: 1px solid $white;
+        color: $white;
+        font-weight: bold;
+        text-align: center;
+      }
+
+      .photodropzone {
+        list-style: none;
+
+        .publisher-photo {
+          float: left;
+          height: 80px;
+          line-height: 80px;
+          list-style: none;
+          margin: 8px;
+          overflow: hidden;
+          vertical-align: middle;
+          width: 80px;
+
+          .x {
+            background-color: $white;
+            border-radius: 25px;
+            color: $black;
+            cursor: pointer;
+            display: none;
+            font-size: 50px;
+            font-weight: bold;
+            height: 50px;
+            left: 15px;
+            line-height: 50px;
+            opacity: .85;
+            position: absolute;
+            text-align: center;
+            top: 15px;
+            width: 50px;
+            z-index: 2;
+
+            &::before {
+              content: '\2716';
+              font-family: 'entypo';
+            }
+          }
+
+          .circle {
+            background-color: $text;
+            border: 1px solid $white;
+            border-radius: 20px;
+            height: 20px;
+            max-height: 20px;
+            max-width: 20px;
+            position: absolute;
+            right: -7px;
+            top: -5px;
+            width: 20px;
+            z-index: 1;
+          }
+
+          &:hover .x {
+            display: inline-block;
+          }
+
+          img {
+            width: 80px;
+          }
+        }
+      }
+
+      .button-container {
+        border-top: 1px solid $light-grey;
+
+        .file-upload {
+          direction: ltr;
+          overflow: hidden;
+          position: relative;
+
+          .file-upload-input {
+            margin: 0;
+            opacity: 0;
+            padding: 0;
+            position: absolute;
+            right: 0;
+            top: 0;
+          }
+        }
+
+        .markdownIndications {
+          color: $medium-gray;
+          line-height: 30px;
```

Comment:
- Nesting should be no greater than 3, but was 4
- Selector should have depth of applicability no greater than 3, but was 5